src/MetricsBuilder.php

```diff
@@ -287,6 +306,101 @@ private function appendAccountIdEndpoint(string $endpoint): void
         }
     }
 
+    /**
+     * Resolves metrics from client arguments.
+     *
+     * @param array $args
+     *
+     * @return void
+     */
+    public function resolveAndAppendFromArgs(array $args = []): void {
```

Comment:
Opening brace should go to a newline.  Also needs a test in `MetricsBuilderTest`